app/uploaders/user_icon_uploader.rb

```diff
@@ -0,0 +1,51 @@
+class UserIconUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  # storage :file
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    if Rails.env.test?
+        Rails.root.join("spec", "support", "uploads", model.class.to_s.underscore, mounted_as, model.id)
```

Comment:
__[RuboCop]__

Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.


<!-- Comment by [SideCI](https://sideci.com) -->